auth/libraries/pip-auth/authlib/validator.py

```diff
@@ -40,6 +40,10 @@ def verify(
             return self.__verify_ticket(token_encoded, cookie_encoded), None
         except jwt.exceptions.ExpiredSignatureError:
             return self.__verify_ticket_from_auth_service(token_encoded, cookie_encoded)
+        except jwt.exceptions.InvalidSignatureError:
+            raise InvalidCredentialsException("The signature of the jwt is invalid")
+        except Exception as e:
```

Comment:
Please do not use a catch-all (antipattern in python), since other errors will not show themselfs anymore (like this one you just fixed).

Reply:
Okay, I removed these lines.